internal/service/rds/consts.go

```diff
@@ -50,6 +50,42 @@ const (
 	EventSubscriptionStatusModifying = "modifying"
 )
 
+const (
+	EngineAurora           = "aurora"
+	EngineAuroraMySQL      = "aurora-mysql"
+	EngineAuroraPostgreSQL = "aurora-postgresql"
+	EngineMySQL            = "mysql"
+	EnginePostgres         = "postgres"
+)
+
+func Engine_Values() []string {
+	return []string{
+		EngineAurora,
+		EngineAuroraMySQL,
+		EngineAuroraPostgreSQL,
+		EngineMySQL,
+		EnginePostgres,
+	}
+}
+
+const (
+	EngineModeGlobal        = "global"
+	EngineModeMultiMaster   = "multimaster"
+	EngineModeParallelQuery = "parallelquery"
+	EngineModeProvisioned   = "provisioned"
+	EngineModeServerless    = "serverless"
```

Comment:
@njoerd114 by any chance.

Do you missed to add the `serverlessv2` value here?
Docs states that we should use that value as engine mode.

Reply:
@jencinar All the testing worked (or at least seemed to 😄) with `provisioned`.

Reply:
should the docs be updated that was not apparent without looking at this comment